.vscode/settings.json

```diff
@@ -33,6 +33,7 @@
     "SDLs",
     "consignable",
     "dataloader",
+    "inquireable",
```

Comment:
Nice! 😄

Reply:
Artsy devs are just out here making up words, I love it 😂